### PR TITLE
Ohie 369 input mapping and validation

### PR DESCRIPTION
**Pull request by bradsawadye**

### CODE REVIEW

File: healthworkforce/mock-services/mcsdMockServices/resources/bundle-Location.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      "fullUrl": "http://hl7.org/fhir/Location/2",
+      "resource": {
+        "resourceType": "Location",
+        "id": "2",
+        "text": {
+          "status": "generated",
+          "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\">USS Enterprise</div>"
+        },
+        "status": "active",
+        "name": "USSS Enterprise-D",
```
Comment (MattyJ007) [inline healthworkforce/mock-services/mcsdMockServices/resources/bundle-Location.json:15]: @bradsawadye Perhaps you could add validation for these fields

File: healthworkforce/mock-services/mcsdMockServices/resources/bundle-Organization.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        {
+          "system": "phone",
+          "value": "+1 555 234 1234",
+          "use": "work"
+        },
+        {
+          "system": "email",
+          "value": "contact@labs.acme.org",
+          "use": "work"
+        }
```
Comment (MattyJ007) [inline healthworkforce/mock-services/mcsdMockServices/resources/bundle-Organization.json:29]: @bradsawadye Please add some validation for these fields. They seem vaguely useful

File: healthworkforce/docker/importer/volume/endpoint.json [hunk 3]
```diff
@@ -0,0 +1,555 @@
...
+          "method": "get",
+          "url": "http://mock-services:4000/ihris-practitionerRole-mock/_history",
+          "headers": {
+            "Content-Type": "application/json"
+          },
+          "params": {
+            "_since": {
+              "path": "state.system.timestamps.lookupRequests.ihris-practitionerRole.requestStart",
+              "prefix": null,
+              "postfix": null
```
[the remark this answers is not in the document]
Comment (bradsawadye) [reply healthworkforce/docker/importer/volume/endpoint.json]: addressed in https://github.com/openhie/instant/pull/43/commits/f0782872f7b5eb6a77a094f367de723bcf4ec570

File: healthworkforce/docker/importer/volume/endpoint.json [hunk 4]
```diff
@@ -0,0 +1,555 @@
...
+          "method": "get",
+          "url": "http://mock-services:4000/ihris-practitioner-mock/_history",
+          "headers": {
+            "Content-Type": "application/json"
+          },
+          "params": {
+            "_since": {
+              "path": "state.system.timestamps.lookupRequests.ihris-practitioner.requestStart",
+              "prefix": null,
+              "postfix": null
```
[the remark this answers is not in the document]
Comment (bradsawadye) [reply healthworkforce/docker/importer/volume/endpoint.json]: addressed in https://github.com/openhie/instant/pull/43/commits/f0782872f7b5eb6a77a094f367de723bcf4ec570

**Pull request merged**